includes/components/class-attribute.php

```diff
@@ -734,6 +734,16 @@ public function add_field_settings( $meta_box ) {
 						'html'       => true,
 						'_order'     => 120,
 					];
+
+					if ( in_array( $field_type, [ 'number', 'text' ], true ) ) {
+						$meta_box['fields'][ $field_context . '_field_default' ] = [
+							'label'  => esc_html__( 'Default Value', 'hivepress' ),
+							'type'   => $field_type,
+							'_order' => 210,
+						];
+					} else {
+						unset( $meta_box['fields'][ $field_context . '_field_default' ] );
```

Comment:
Is unsetting needed for some AJAX refreshing? It seems that the condition above prevents adding the field already so unsetting may not be required